### PR TITLE
add onTouchStart

### PR DESCRIPTION
**Pull request by christianwen**
### Details

Add onTouchStart to MarkdownTextInput

### Related Issues

GH_LINK
https://github.com/Expensify/App/issues/10731

### CODE REVIEW

File: src/MarkdownTextInput.web.tsx [hunk 1]
```diff
@@ -590,6 +592,14 @@ const MarkdownTextInput = React.forwardRef<TextInput, MarkdownTextInputProps>(
       divRef.current = r as MarkdownTextInputElement;
     };
 
+    const handleTouchStart = (event: TouchEvent<HTMLDivElement>) => {
+      if (!onTouchStart) {
+        return;
+      }
+      const e = event as unknown as GestureResponderEvent;
+      onTouchStart(e);
```
Comment (tomekzaw) [inline src/MarkdownTextInput.web.tsx:600]:
```suggestion
      onTouchStart?.(event as unknown as GestureResponderEvent);
```

**Review by tomekzaw — APPROVED** [review]
LGTM, let's wait for @Skalakid's review prior to merging

File: src/MarkdownTextInput.web.tsx (hunk 1 above)
Comment (tomekzaw) [reply src/MarkdownTextInput.web.tsx:600]: Thanks for response!

**Review by BartoszGrajdek — APPROVED** [review]
LGTM, WDYT @Skalakid?

**Pull request merged**